core/new-gui/src/app/workspace/component/property-editor/property-editor.component.ts

```diff
@@ -65,18 +74,27 @@ export class PropertyEditorComponent implements OnInit {
           .getCurrentHighlightedOperatorIDs();
         const highlightedGroups = this.workflowActionService.getJointGraphWrapper().getCurrentHighlightedGroupIDs();
         const highlightLinks = this.workflowActionService.getJointGraphWrapper().getCurrentHighlightedLinkIDs();
+        const workflowVersions = this.workflowVersionService.getWorkflowVersions();
 
-        if (highlightedOperators.length === 1 && highlightedGroups.length === 0 && highlightLinks.length === 0) {
+        if (workflowVersions.length > 0 && highlightLinks.length === 0 && highlightedGroups.length === 0 && highlightedOperators.length === 0) {
```

Comment:
I don't think this is a good condition to decide to switch components, because even when the user is not highlighting any operator nor the version text, the verion list is still shown. Maybe it's better to add a `versionHighlighted` variable somewhere